frontend/src/metabase/reference/guide/GettingStartedGuide.jsx

```diff
@@ -0,0 +1,296 @@
+    important_segments && important_segments.length !== 0 ? false :
+    important_tables && important_tables.length !== 0 ? false :
+    true;
+
+const mapStateToProps = (state, props) => ({
+    guide: getGuide(state, props),
+    user: getUser(state, props),
+    dashboards: getDashboards(state, props),
+    metrics: getMetrics(state, props),
+    segments: getSegments(state, props),
+    tables: getTables(state, props),
+    // FIXME: avoids naming conflict, tried using the propNamespace option
+    // version but couldn't quite get it to work together with passing in
+    // dynamic initialValues
+    metadataFields: getFields(state, props),
+    loading: getLoading(state, props),
+    // naming this 'error' will conflict with redux form
+    loadingError: getError(state, props),
+    isEditing: getIsEditing(state, props),
+});
+
+const mapDispatchToProps = {
+    updateDashboard,
+    createDashboard,
+    updateSetting,
+    clearRequestState,
+    ...metadataActions,
+    ...actions
+};
+
+@connect(mapStateToProps, mapDispatchToProps)
+export default class GettingStartedGuide extends Component {
+    static propTypes = {
+        fields: PropTypes.object,
+        style: PropTypes.object,
+        guide: PropTypes.object,
+        user: PropTypes.object,
+        dashboards: PropTypes.object,
+        metrics: PropTypes.object,
+        segments: PropTypes.object,
+        tables: PropTypes.object,
+        metadataFields: PropTypes.object,
+        loadingError: PropTypes.any,
+        loading: PropTypes.bool,
+        startEditing: PropTypes.func,
+    };
+
+    render() {
+        const {
+            style,
+            guide,
+            user,
+            dashboards,
+            metrics,
+            segments,
+            tables,
+            metadataFields,
+            loadingError,
+            loading,
+            startEditing,
+        } = this.props;
+
+        return (
+            <div className="full relative py4" style={style}>
+                <LoadingAndErrorWrapper className="full" style={style} loading={!loadingError && loading} error={loadingError}>
+                { () => 
+                    <div>
+                        <GuideHeader
+                            startEditing={startEditing}
+                            isSuperuser={user && user.is_superuser}
+                        />
+
+                        <div className="wrapper wrapper--trim">
+                            { (!guide || isGuideEmpty(guide)) && user && user.is_superuser && (
+                                <AdminInstructions>
+                                    <h2 className="py2">Help your team get started with your data.</h2>
+                                    <GuideText>
+                                        Show your team what’s most important by choosing your top dashboard, metrics, and segments.
+                                    </GuideText>
+                                    <button
```

Comment:
@kdoh if I split up the routes, then this button should take you to `/reference/edit`. What's the usual way of doing this? An <a> or changing the startEditing function to go to that url?